### ISSUE TITLE
Change Z_CONFIG_USE_TRANSACTIONS to 1 in config.php 

### ISSUE THREAD

**Issue by robertgdev** [zikula/core#796]
I have no idea why this was not done earlier, but at this point in time there really is no case for not using transactions in the default. MySql supports them (on InnoDB) and other databases (should we ever again officially support them) also do. 

Not only that, for some operations using transactions are actually faster (lots of inserts, for example) and in the case of complex multi-statement operations they provide security against having an inconsistent database state in case a failure occours 1/2 way through the operation. 

In short, please make the use of transactions the default. There should be no downside to this but significant upsides.
